app/views/admin/contacts/_tab_heading.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="govuk-body">
+  <%= render "govuk_publishing_components/components/summary_list", {
+    title: title_text,
+    edit: if (contactable.home_page_contacts.many? && contacts.none?(&:foi?))
+            {
+              href: reorder_admin_organisation_contacts_path,
+              link_text: "Reorder",
+            }
+          else
+            {}
+          end
```

Comment:
I know we've done it in other places but if you need less space between here and the tabs we can do something like

```
<div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds">
      <%= render "govuk_publishing_components/components/heading", {
          text: title_text
       } %>      
  </div>
  <% if if (contactable.home_page_contacts.many? && contacts.none?(&:foi?)) %>
    <div class="govuk-grid-column-one-third">
      <p class="govuk-body govuk-!-text-align-right"> 
         <%=  link_to("Reorder",  href: reorder_admin_organisation_contacts_path, class: "govuk-link" %>
       </p>
    </div>
  <% end %>
</div>
```